infrastructure/k8s/values.schema.json

```diff
@@ -7,6 +7,10 @@
       "description": "The name of this deployment of Taskcluster.",
       "type": "string"
     },
+    "imagePullSecrets": {
+      "description": "Secrets to get the docker image",
```

Comment:
It isn't clear to me what the format of each secret is, and how they are used to return the docker image, and which  docker image the description refers to (although I guess it is the one and only image that is shared across all the service containers). Would it be ok to add a couple of words to explain how the secrets are used to retrieve the docker image, and just clarify which docker image with something like "the taskcluster services docker image"? Thanks @Eijebong !

Reply:
This is from https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry

Reply:
Format there is indeed an array, but elements should be  a dict with `name` key present. Would be good to specify it, indeed.

```yaml
imagePullSecrets:
   - name: secret name
```

How often do we need more than one secret name? 

I guess, one way to avoid using schema is to use single entry like:

```yml
      imagePullSecrets:
        - name: {{ .Values.imagePullSecrets }}
```   
Wdyt?